packages/xrpl/src/models/transactions/permissionedDomainSet.ts

```diff
@@ -0,0 +1,46 @@
+import { isArray } from 'lodash'
```

Comment:
We shouldn't use lodash

Reply:
is it because of low-frequency of recent commits? Do you have any alternative preferences?

Reply:
https://youmightnotneed.com/lodash

Reply:
lodash is a massive package and we don't want that size bloat in the xrpl.js package. https://bundlephobia.com/package/lodash@4.17.21

Reply:
okay. However, the size really matters only in memory-constrained environments right? Are any applications affected by the large size of the xrpl.js application ?

Reply:
[2697a21](https://github.com/XRPLF/xrpl.js/pull/2874/commits/2697a218a1e8bb65e9099eaa6857e9f9b43a27dc)

Reply:
In JS, memory always matters, because it'll affect a lot of things when you're running the code in the browser.